projects/kit/components/input-password/input-password-options.ts

```diff
@@ -0,0 +1,28 @@
+import {InjectionToken} from '@angular/core';
+import {TuiContextWithImplicit} from '@taiga-ui/cdk';
+import {TuiSizeL, TuiSizeS} from '@taiga-ui/core/types';
```

Comment:
Cross-package imports should be from the main entry point

Reply:
Done. Can you please explain why?

Reply:
Since all our lower-level entry points are re-exported again on the higher level you get the same treeshakability if you import from the top level. When in the same library we write exact secondary entry points so that there are no circular dependencies and each entry point can clearly resolve its dependencies.

Reply:
Thank you for the explanation